[12/Jun/2017 13:40:34] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:42:56] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:43:08] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:43:37] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:43:52] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:44:03] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:44:26] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:44:39] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:45:01] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:46:04] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:46:13] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:46:37] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:46:56] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:48:36] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:53:53] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:54:21] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:54:44] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:55:37] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:56:01] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:56:28] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:57:04] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:57:28] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:57:47] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:58:26] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[12/Jun/2017 13:59:50] ERROR [django.request:256] Internal Server Error: /task/5/clueweb12-0303wb-67-14408.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 03/clueweb12-0303wb-67-14408.html
[13/Jun/2017 17:24:00] ERROR [django.request:256] Internal Server Error: /task/
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/user_system/utils.py", line 70, in ret
    return func(*args)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 23, in assessment
    return render(request, 'clueweb12-0404wb-24-04919.html',locals())
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 35, in get_template
    return engine.get_template(template_name, dirs)
  File "/Library/Python/2.7/site-packages/django/template/backends/django.py", line 30, in get_template
    return Template(self.engine.get_template(template_name, dirs))
  File "/Library/Python/2.7/site-packages/django/template/engine.py", line 167, in get_template
    template, origin = self.find_template(template_name, dirs)
  File "/Library/Python/2.7/site-packages/django/template/engine.py", line 141, in find_template
    source, display_name = loader(name, dirs)
  File "/Library/Python/2.7/site-packages/django/template/loaders/base.py", line 13, in __call__
    return self.load_template(template_name, template_dirs)
  File "/Library/Python/2.7/site-packages/django/template/loaders/base.py", line 17, in load_template
    template_name, template_dirs)
  File "/Library/Python/2.7/site-packages/django/template/loaders/app_directories.py", line 39, in load_template_source
    return fp.read(), filepath
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/codecs.py", line 302, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xa9 in position 42032: invalid start byte
[13/Jun/2017 17:49:28] ERROR [django.request:256] Internal Server Error: /task/1/clueweb12-0206wb-80-04257.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 02/clueweb12-0206wb-80-04257.html
[13/Jun/2017 17:49:55] ERROR [django.request:256] Internal Server Error: /task/1/clueweb12-0206wb-80-04257.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 02/clueweb12-0206wb-80-04257.html
[13/Jun/2017 17:50:08] ERROR [django.request:256] Internal Server Error: /task/1/clueweb12-0206wb-80-04257.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 02/clueweb12-0206wb-80-04257.html
[13/Jun/2017 18:17:03] ERROR [django.request:256] Internal Server Error: /task/1/clueweb12-0206wb-80-04257.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 02/clueweb12-0206wb-80-04257.html
[13/Jun/2017 18:17:15] ERROR [django.request:256] Internal Server Error: /task/1/clueweb12-0206wb-80-04257.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 122, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 02/clueweb12-0206wb-80-04257.html
[16/Aug/2017 18:26:55] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-0203wb-85-11564.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 02/clueweb12-0203wb-85-11564.html
[16/Aug/2017 18:27:01] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-0208wb-28-26611.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 02/clueweb12-0208wb-28-26611.html
[16/Aug/2017 18:27:06] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-1008wb-40-28539.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 10/clueweb12-1008wb-40-28539.html
[16/Aug/2017 18:27:07] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-0001wb-13-01078.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 00/clueweb12-0001wb-13-01078.html
[16/Aug/2017 18:27:21] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-1100wb-38-18940.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 11/clueweb12-1100wb-38-18940.html
[16/Aug/2017 18:27:23] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-0001wb-13-01078.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 00/clueweb12-0001wb-13-01078.html
[16/Aug/2017 18:27:26] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-1100wb-38-18940.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 11/clueweb12-1100wb-38-18940.html
[16/Aug/2017 18:27:29] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-0001wb-13-01078.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 00/clueweb12-0001wb-13-01078.html
[16/Aug/2017 18:27:30] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-1100wb-38-18940.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 11/clueweb12-1100wb-38-18940.html
[16/Aug/2017 18:27:33] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-0203wb-78-20890.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 02/clueweb12-0203wb-78-20890.html
[16/Aug/2017 18:28:40] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-1104wb-01-03443.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 11/clueweb12-1104wb-01-03443.html
[16/Aug/2017 18:28:46] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-1717wb-05-05385.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 17/clueweb12-1717wb-05-05385.html
[16/Aug/2017 18:28:49] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-1217wb-42-14789.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 12/clueweb12-1217wb-42-14789.html
[16/Aug/2017 18:28:52] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-1104wb-01-03443.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 11/clueweb12-1104wb-01-03443.html
[16/Aug/2017 18:30:58] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-1217wb-42-14789.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 12/clueweb12-1217wb-42-14789.html
[16/Aug/2017 18:31:02] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-1805wb-75-02649.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 18/clueweb12-1805wb-75-02649.html
[16/Aug/2017 18:31:03] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-0208wb-42-00685.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 02/clueweb12-0208wb-42-00685.html
[16/Aug/2017 18:31:04] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-0812wb-17-18520.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 08/clueweb12-0812wb-17-18520.html
[16/Aug/2017 18:31:06] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-1104wb-01-03443.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 11/clueweb12-1104wb-01-03443.html
[16/Aug/2017 18:31:09] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-1717wb-05-05385.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 17/clueweb12-1717wb-05-05385.html
[16/Aug/2017 18:31:12] ERROR [django.request:256] Internal Server Error: /task/12/clueweb12-1217wb-42-14789.html
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/Fan/Downloads/NTCIRMDB2/assessment/views.py", line 242, in show
    return render(request, a+"/"+url)
  File "/Library/Python/2.7/site-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: 12/clueweb12-1217wb-42-14789.html
